code/modules/surgery/plastic_surgery.dm

```diff
@@ -4,10 +4,32 @@
 	steps = list(
 		/datum/surgery_step/incise,
 		/datum/surgery_step/retract_skin,
+		/datum/surgery_step/insert_plastic,
 		/datum/surgery_step/reshape_face,
 		/datum/surgery_step/close,
 	)
 
+//insert plastic
+/datum/surgery_step/insert_plastic
+	name = "insert plastic (plastic)"
+	implements = list(
+		/obj/item/stack/sheet/plastic = 100,
+		/obj/item/food/meat/slab = 100)
```

Comment:
this won't ever try to feed the slab to the victim, correct?

Reply:
correct, I double-checked